balancer/rls/internal/child_policy.go

```diff
@@ -0,0 +1,216 @@
+/*
+ *
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package rls
```

Comment:
This needs to be `package internal` IMO, or be in `rls/internal/rls` or `rls/`.  Otherwise:

```go
import "grpc/balancer/rls/internal"

rls.Foo() // where was rls defined???
```

Reply:
That's fine, but please make sure whenever you `import rls/internal`, you rename the import to `rls` explicitly.  (`import rls "grpc/balancer/rls/internal"`)